contrib/inventory/scaleway.ini

```diff
@@ -0,0 +1,34 @@
+# Ansible dynamic inventory script for Scaleway cloud provider
+#
+
+[compute]
+# Fetch inventory for regions. If not defined will read the SCALEWAY_REGION environment variable
+#
+# regions = all
+# regions = ams1
+# regions = par1, ams1
+regions = par1
+
+
+# Define a Scaleway token to perform required queries on the API
+# in order to generate inventory output.
+#
+[auth]
+api_token = mysecrettoken
+
+
+# To avoid performing excessive calls to Scaleway API you can define a
+# cache for the plugin output. Within the time defined in seconds, latest
+# output will be reused. After that time, the cache will be refreshed.
+#
+[cache]
+cache_max_age = 60
+cache_dir = /tmp
```

Comment:
That would result into insecure default, since someone would be able to inject a cache file in a shared server. This would also fail if there is concurent run of ansible using the same inventory.

Reply:
Indeed. I've fixed that, thanks!